help/edit-a-message.md

```diff
@@ -50,6 +50,23 @@ of a message.
     [view a message's edit history](/help/view-a-messages-edit-history)
     if it is [enabled](/help/disable-message-edit-history) in your organization.
 
+## Message notifications
+
+If you edit a message to add a mention of a user or group, the newly mentioned
+users will receive notifications just as if they had been mentioned in the
+original message.
+
+If you edit a message soon after sending it, the edit will be reflected in the
+content of any [email notifications that have not yet been
+sent](https://zulip.com/help/email-notifications#delay-before-sending-emails),
+including canceling notifications for users whose @-mention was converted from a
```

Comment:
Same notes as above:
- Can be a relative link here.
- Question about the use of "@-mention" instead of just "mention".